src/main/java/uk/gov/justice/digital/hmpps/whereabouts/repository/VideoLinkBookingRepository.kt

```diff
@@ -2,15 +2,29 @@ package uk.gov.justice.digital.hmpps.whereabouts.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.query.Param
+import uk.gov.justice.digital.hmpps.whereabouts.model.HearingType
 import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkBooking
 
 interface VideoLinkBookingRepository : JpaRepository<VideoLinkBooking, Long> {
   @Query(
     """
```

Comment:
This query avoids N + 1 select problem.  Using `b.id` and `a.videoLinkBooking.id` instead of `b` and `a.videoLinkBooking` gives cleaner, simpler SQL.